Sources/SwiftParserDiagnostics/ParserDiagnosticMessages.swift

```diff
@@ -691,6 +694,9 @@ extension FixItMessage where Self == StaticParserFixIt {
   public static var wrapInBackticks: Self {
     .init("if this name is unavoidable, use backticks to escape it")
   }
+  public static var convertToExtendedRegexLiteral: Self {
+    .init("convert to extended regex literal")
```

Comment:
I’m not sure if *extended regex literal* is a term that most developers know. What do you think of changing this to

```suggestion
    .init("convert to extended regex literal with '#'")
```